apps/admin/backend/src/store.ts

```diff
@@ -1725,6 +1725,11 @@ export class Store {
           left join
             write_in_candidates on write_in_candidates.id = write_ins.write_in_candidate_id
           where ${whereParts.join(' and ')}
+            and (
+              write_ins.is_unmarked = 0 or
+              official_candidate_id is not null or
+              write_in_candidate_id is not null
+            )
```

Comment:
I'm having some trouble understanding what this new clause does. Not sure if `write_ins.is_unmarked = 0` is filtering unmarked write-ins in or out (is 0 true or false?). Also not sure what it means when `official_candidate_id` or `write_in_candidate_id` are not null. Maybe add an explanatory comment?

Reply:
Yeah, the adjudication logic is difficult to parse. I will add a comment.